packages/terra-consumer-site/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+Changelog
+=========
```

Comment:
You'll want to clean this up / update this when you get ready to release terra-consumer-site